docs/developers/coredev/release.md

```diff
@@ -2,50 +2,61 @@
 
 # Release guide
 
-This guide documents `napari`'s release process. Most required tools mentioned here are in https://github.com/napari/napari-release-tools
+This guide documents `napari`'s release process as of 0.6.0.
+Most required tools mentioned here are in https://github.com/napari/napari-release-tools.
 
-# Timeline
+## Timeline
 
-Currently, we are releasing a new napari version every 2-3 weeks. Release candidates are made available as a reasonableness check for 1-24h before the full release is published: since the release cadence is so high, any new bugs can be quickly rectified, and more quickly identified by making a full release.
+Currently, we are releasing a major version every six to twelve months.
+We plan minor napari version releases every 3 months.
+We will ship bug fix releases as needed.
 
-When major API changes and deprecations happen, we will provide longer release candidate cycles to ensure proper testing by the community.
+Prior to major and minor releases, we will have a release testing cycle with
+alpha prereleases and release candidates. All will be made available to the
+community for testing. The length of time between these prereleases will
+depend on the size of the release and feedback received. A conservative estimate
+is one week between each prerelease.
 
-The latest release candidate can be installed with
+The latest prerelease (alpha or release candidate) can be installed with:
 
 `python -m pip install --pre napari`
 
-# Release management
+`uv pip install --pre napari`
+
```

Comment:
```suggestion

:::{important}
Note: the `--pre` in the above commands will install prereleases of napari and all dependencies! This may result in errors or warnings. If this happens, you can instead install napari without `--pre`, but instead specify `a0` in the installation. For example, for a prerelease of a hypothetical `1.0.0` version of napari, you would install `"napari>=1.0.0a0"`. Replace `1.0.0` with the actual version you are interested in.
:::
```